### PR TITLE
Navigate API

### PR DESCRIPTION
**Pull request by msgerbush**
Putting this out there for feedback.  Not sure if we want to take this approach, or just allow the developer to set `window.parent.location` directly, which we can allow from within the sandbox.  

We could also allow for navigating to specific resources directly, rather than routes, so something like `card.services('helpdesk').trigger('ticket:navigate', 2)`.

### CODE REVIEW

File: docs/apis/environment.md [hunk 1]
```diff
@@ -258,3 +258,20 @@ Example user (note all arrays have been reduced to a single example item)
...
+
+```js
+card.services('environment').trigger('navigate', path)
+```
+
+##### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`path`|`string`| The relative path for the host application to visit
```
Comment (babbtx) [inline docs/apis/environment.md]: I think it warrants an example or something so that devs know where to get the relative URL for resources.

**Comment by msgerbush** [comment]
Agreed on the example.  What do you think of the overall approach?  Does this make more sense than linking to resources?  Should we just enable the frame to do top navigation on `window.location`?

**Comment by babbtx** [comment]
I like the approach. I think it constrains it so that when we have the same APIs available in other apps (hosted helpdesk, network monitor) they're consistent. `card.trigger('navigate', resource['show_url'])`.

**Comment by msgerbush** [comment]
Updated to remove docs.  Ready to merge when you are.

**Pull request merged**